.github/workflows/static.yaml

```diff
@@ -36,3 +36,7 @@ jobs:
     - name: Run pyflakes
       run: |
         tox -e pyflakes
+
+    - name: Run docs
+      run: |
+        tox -e docs
```

Comment:
I can remove this from this PR and add it later...

As a first run, I wanted to see that Sphinx docs are failing on errors.